scripts/bin/create.py

```diff
@@ -19,13 +17,13 @@
 def required_environment_directory(environment_variable, description_for_error):
     directory = os.environ.get(environment_variable, None)
     if not directory:
-        print "'%s' environment variable is required. You can add this to your ~/.bash_profile by adding the line %s=[%s]" % (
-            environment_variable, environment_variable, description_for_error)
+        print("'%s' environment variable is required. You can add this to your ~/.bash_profile by adding the line %s=[%s]" % (
```

Comment:
Not sure if it is worth it for this, but FYI Python 3.6+ has "f-strings" which means you can do interpolation a little like scala: https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python

Could change this to

    print(f"'{environmental_variable}' environment variable is required. You can ....